fast64_internal/utility.py

```diff
@@ -164,18 +164,30 @@ def checkObjectReference(obj, title):
         )
 
 
-def selectSingleObject(obj: bpy.types.Object):
-    bpy.ops.object.select_all(action="DESELECT")
+def getActiveObject():
+    return bpy.context.view_layer.objects.active
```

Comment:
is this ``bpy.types.Object`` by default or is it ``Any``? if it's not ``Object`` you should add the return type hint stuff (I mean this: ``def getActiveObject() -> bpy.types.Object``)